src/main/java/seedu/parser/MainParser.java

```diff
@@ -74,7 +75,14 @@ private Command parseAddContact(String userInput) {
             if (details[NAME_INDEX] == null) {
                 throw new MissingNameException();
             }
-            return new AddContactCommand(details[DetailType.NAME.getIndex()], details[DetailType.GITHUB.getIndex()]);
+            assert details.length == NUMBER_OF_FIELDS;
+            String name = details[DetailType.NAME.getIndex()];
+            String github = details[DetailType.GITHUB.getIndex()];
+            String linkedin = details[DetailType.LINKEDIN.getIndex()];
+            String telegram = details[DetailType.TELEGRAM.getIndex()];
+            String twitter = details[DetailType.TWITTER.getIndex()];
+            String email = details[DetailType.EMAIL.getIndex()];
```

Comment:
All details will be added as `null` here if not specified. 

@ashrafjfr please take note of this and handle accordingly for view contact